### PR TITLE
Adds Priority and Pod Preemption to Cluster Autoscaled FAQ.

### PR DESCRIPTION
**Pull request by krzysztof-jastrzebski**

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -182,6 +183,29 @@ No. CPU-based (or any metric-based) cluster/node group autoscalers, like
 [GCE Instance Group Autoscaler](https://cloud.google.com/compute/docs/autoscaler/), are NOT compatible with CA.
 They are also not particularly suited to use with Kubernetes in general.
 
+
+### How Cluster Autoscaler works with Pod Priority and Preemption?
+
+Pod Priority and Preemption feature enables K8S users to schedule pods based on priorities if there is not enough resources.
```
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: Add information about from which version will this be handled in this way and how it was handled  before.

Comment (krzysztof-jastrzebski) [reply cluster-autoscaler/FAQ.md]: done

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -182,6 +183,33 @@ No. CPU-based (or any metric-based) cluster/node group autoscalers, like
...
+Pods with priority lower than cut off:
+* don't trigger scale ups, no new node is added in order to run them,
+* don't prevent scale downs, nodes running such pods can be deleted.
+
+Nothing changes for Pods with priority greater or equal to cut off and pods without priority.
+
+Default priority cut off is 0. It can be changed using flag but we discourage it.
+Cluster Autoscaler also don't trigger scale up if an unschedulable pod is waiting for lower priority pod preemption.
+
+Prior 1.9 CA didn't take priorities into account.
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: I think s/Prior/Prior to/

File: cluster-autoscaler/FAQ.md [hunk 3]
```diff
@@ -182,6 +183,33 @@ No. CPU-based (or any metric-based) cluster/node group autoscalers, like
...
+On the other hand Cluster Autoscaler makes sure that there is enough resources to run all pods.
+We introduced priority cut off to Cluster Autoscaler which tells which pods should trigger Cluster Autoscaler actions.
+
+Pods with priority lower than cut off:
+* don't trigger scale ups, no new node is added in order to run them,
+* don't prevent scale downs, nodes running such pods can be deleted.
+
+Nothing changes for Pods with priority greater or equal to cut off and pods without priority.
+
+Default priority cut off is 0. It can be changed using flag but we discourage it.
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: If you mention there is a flag you may as well put it's name here.

**Comment by mwielgus** [comment]
I didn't notice Maciek's comments. Please fix them in a separate PR.

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (krzysztof-jastrzebski) [reply cluster-autoscaler/FAQ.md]: Done

**Pull request merged**